src/Symfony/Component/Cache/composer.json

```diff
@@ -31,7 +31,7 @@
     "require-dev": {
         "cache/integration-tests": "dev-master",
         "doctrine/cache": "~1.6",
-        "doctrine/dbal": "~2.5",
+        "doctrine/dbal": "~2.5@stable",
```

Comment:
`DriverManager::getConnection(['url' =>])` has been introduced in `2.5.0` (does not exists in BETAs, nor RCs) https://github.com/doctrine/dbal/commit/e8056f53024f96b95dc09ade256a2f5fb8a5e124